pkg/serverless/daemon/routes.go

```diff
@@ -71,10 +71,17 @@ func (e *EndInvocation) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	log.Debug("Hit on the serverless.EndInvocation route.")
 	endTime := time.Now()
 	ecs := e.daemon.ExecutionContext.GetCurrentState()
+	responseBody, err := ioutil.ReadAll(r.Body)
```

Comment:
I think we might want to add a customer-configurable timeout while reading this?
For huge payload maybe we don't want to add extra latency?